std_msgs/Header header
float64 cam_alt_reader          # cam_alt_reader reading
float64 cam_alt_reader_var      # cam_alt_reader_var reading
